src/@chakra-ui/gatsby-plugin/theme.ts

```diff
@@ -44,4 +44,5 @@ const theme: ThemeOverride = {
   components,
 }
 
+// TODO: use `extendBaseTheme` when updating theming for components for new DS
```

Comment:
Whats the difference between `extendBaseTheme` vs `extendTheme`. Couldn't find it on the docs.